packages/bpk-component-modal/src/themeAttributes-test.js

```diff
@@ -0,0 +1,32 @@
+/*
+  * Backpack - Skyscanner's Design System
+  *
+  * Copyright 2017 Skyscanner Ltd
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *   http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+import themeAttributes from './themeAttributes';
+
+describe('themeAttributes', () => {
+  it('should export the correct theme attributes', () => {
+    expect(themeAttributes).toEqual([
+      'linkColor',
```

Comment:
This looks like copypasta - link stuff shouldn't be in here right?

Reply:
Or if it should, then it needs to be added to the readme?

Reply:
is not an error, the `themeAttributes`from the `bpk-component-link` have been added to theme the type of modal with the closing button as link